pkg/diagnose/deployments.go

```diff
@@ -172,7 +172,9 @@ func checkDaemonset(k8sClient kubernetes.Interface, namespace, daemonSetName str
 }
 
 func checkPodsStatus(k8sClient kubernetes.Interface, namespace string, status reporter.Interface) {
-	pods, err := k8sClient.CoreV1().Pods(namespace).List(context.TODO(), metav1.ListOptions{})
+	pods, err := k8sClient.CoreV1().Pods(namespace).List(context.TODO(), metav1.ListOptions{
+		LabelSelector: "transient!=true",
```

Comment:
We should qualify the label name to be safe.

```suggestion
		LabelSelector: "submariner.io/transient!=true",
```